spec/lib/reportsv2_spec.rb

```diff
@@ -237,7 +237,7 @@
       end
     end
 
-    context 'XLS reports' do
+    context 'XLS reports', :pro_account do
```

Comment:
I think you've done a good work with this repo.

But I'm not really sure how to go further with this pull request.
Currently tests for original master branch have some more failures. 
For example, https://travis-ci.org/kanet77/togglv8/jobs/202525485
It seems that the data on test toggl account has been changed (e.g. there are 2 users there now, although the test expects only one). I can try to fix these tests in the next commit for this pull request, but should I?